pkg/clusteragent/admission/mutate/cwsinstrumentation/k8scp/cp.go

```diff
@@ -0,0 +1,184 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build kubeapiserver
+
+// Package k8scp implements the necessary methods to copy a local file to a remote
+// container
+package k8scp
+
+import (
+	"archive/tar"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	"k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/kubectl/pkg/scheme"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// Copy perform remote copy operations
+type Copy struct {
+	Container string
+	Namespace string
+
+	config    *restclient.Config
+	clientSet kubernetes.Interface
```

Comment:
I cannot find any reference to `.clientSet`? Am I missing something?

Reply:
Likely a legacy piece of code from one of the unsuccessful attempts at implementing this feature ! Nice catch 👍🏻